packages/unmock-jsdom/src/persistence/local-storage-persistence.ts

```diff
@@ -1,29 +1,30 @@
-import * as yml from "js-yaml";
 import * as path from "path";
-import { IPersistableData, IPersistence } from "unmock-core";
+import { IMetaData, IPersistence, IRequestData, IResponseData } from "unmock-core";
 
 // Uses directory structure for keys in window local storage
-const UNMOCK_KEY = ".unmock";
+const UNMOCK_FILE = ".unmock";
```

Comment:
`s/_FILE/_KEY/g`
This is still in browser, there are no files, are there?

Reply:
No, but it was originally named `UNMOCK_FILE`, so I changed it back to that. The reason IIRC was to have the same naming convention everywhere.